src/overlays/actors/ovl_Bg_Spot18_Basket/z_bg_spot18_basket.h

```diff
@@ -6,11 +6,34 @@
 
 struct BgSpot18Basket;
 
+typedef void (*BgSpot18BasketActionFunc)(struct BgSpot18Basket*, GlobalContext*);
+
+typedef struct D_808B85E4Struct {
+    s16 unk_0;
+    s16 unk_2;
+    s16 unk_4;
+} D_808B85E4Struct;
+
 typedef struct BgSpot18Basket {
-    /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0xD0];
+    /* 0x0000 */ DynaPolyActor dyna;
+    /* 0x0164 */ ColliderJntSph colliderJntSph;
+    /* 0x0184 */ ColliderJntSphItem colliderJntSphItem;
+    /* 0x01C4 */ char unk_1C4[0x40];
```

Comment:
```suggestion
    /* 0x0184 */ ColliderJntSphItem colliderJntSphItems[2];
```
this is coherent with the size (2) of `static ColliderJntSphItemInit sJntSphItemsInit[2]`